lib/index.js

```diff
@@ -66,6 +66,7 @@ module.exports = (opts, ctx) => {
 
       Yuque.setRepoId(repoId)
       const yuque = Yuque.getInstance()
+      yuque.repoId = repoId
```

Comment:
Actually this reassigning doesn't solve this problem from the root, you also need update the `store` in yuque instance.